### PR TITLE
Add training weights and readme file of sknet

### PR DESCRIPTION
**Pull request by Silver0930**
Thank you for your contribution to the MindCV repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

The training recipe and trained weights of sknet are added.

## Test Plan

Please refer to "configs/sknet/README.md" for reproducing and testing.

## Related Issues and PRs

#437

### CODE REVIEW

File: configs/sknet/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  <em>Figure 1. Selective Kernel Convolution.</em>
+</p>
+
+## Results
+
+<div align="center">
+
+| Model          | Context | Top-1 (%) | Top-5 (%) | Params (M) | Recipe                                                                                               | Download                                                                                    |
+|----------------|---------|-----------|-----------|------------|------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------|
+| skresnet18 | D910x8-G | 73.09 | 91.2 | 11.97 | [yaml](https://github.com/mindspore-lab/mindcv/blob/main/configs/sknet/skresnet18_ascend.yaml) | [weights](https://download.mindspore.cn/toolkits/mindcv/sknet/sknet18-868228e5.ckpt) |
```
Comment (Songyuanwei) [inline configs/sknet/README.md:28]: 保留小数点后两位

File: configs/sknet/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+```
+
+### Deployment
+
+To deploy online inference services with the trained model efficiently, please refer to the [deployment tutorial](https://github.com/mindspore-lab/mindcv/blob/main/tutorials/deployment.md).
+
+
+## References
+<!--- Guideline: Citation format GB/T 7714 is suggested. -->
+[1] D. H. Hubel and T. N. Wiesel. Receptive fields, binocular interaction and functional architecture in the cat’s visual
```
Comment (Songyuanwei) [inline configs/sknet/README.md:90]: References 混在一起了
image

File: configs/sknet/skresnet18_ascend.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+epoch_size: 200
+dataset_sink_mode: True
+amp_level: "O2"
+
+# loss
+loss: "CE"
+label_smoothing: 0.1
+
+# lr scheduler
+scheduler: "warmup_cosine_decay"
```
Comment (Songyuanwei) [inline configs/sknet/skresnet18_ascend.yaml:39]: 参数更正 "warmup_cosine_decay" -> "cosine_decay"

File: configs/sknet/skresnet34_ascend.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+epoch_size: 250
+dataset_sink_mode: True
+amp_level: "O2"
+
+# loss
+loss: "CE"
+label_smoothing: 0.1
+
+# lr scheduler
+scheduler: "warmup_cosine_decay"
```
Comment (Songyuanwei) [inline configs/sknet/skresnet34_ascend.yaml:41]: 参数更正 "warmup_cosine_decay" -> "cosine_decay"

File: configs/sknet/skresnext50_32x4d_ascend.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+epoch_size: 200
+dataset_sink_mode: True
+amp_level: "O2"
+
+# loss
+loss: "CE"
+label_smoothing: 0.1
+
+# lr scheduler
+scheduler: "warmup_cosine_decay"
```
Comment (Songyuanwei) [inline configs/sknet/skresnext50_32x4d_ascend.yaml:39]: 参数更正 "warmup_cosine_decay" -> "cosine_decay"

**Review by geniuspatrick — APPROVED** [review]

**Pull request merged**